### PR TITLE
Prepare 5.0.0

### PR DESCRIPTION
**Pull request by leplatrem**
**Step 1**
- [ ] Merge remaining pull requests
- [x] Update changelog
- [x] Update version in docs/conf.py
- [x] Version dependencies
- [x] If the protocol was updated, upgrade API protocol in `docs/api/index.rst` and mention the new protocol version in `CHANGELOG.rst`
- [x] Update release date in `docs/api/index.rst`

**Step 2**
- [ ] Tag vX.Y.Z
- [ ] Publish on Pypi

**Step 3**
- [ ] Close milestone in Github
- [ ] Add entry in Github release page
- [ ] Create next milestone in Github
- [ ] Configure the version in ReadTheDocs
- [ ] Check that a Docker image was built
- [ ] Send mail to ML
- [ ] Tweet!
- [ ] Deploy new version on demo server
- [ ] Upgrade dependency in `kinto-dist` repo
- [ ] Upgrade version targetted in `kinto-heroku` repo
- [ ] Upgrade version of Kinto server for the tests of clients and plugins repos
  (_kinto-http.js, kinto-http.py, kinto-attachment, etc._)

@Natim  r?

### CODE REVIEW

File: kinto/plugins/admin/__init__.py [hunk 1]
```diff
@@ -7,7 +7,7 @@ def includeme(config):
     # Expose capability.
     config.add_api_capability(
         "admin",
-        version="1.4.3",
+        version="1.4.3 invalid",
```
Comment (Natim) [inline kinto/plugins/admin/__init__.py]: 1.5.0?

**Comment by leplatrem** [comment]
Just update, take over if you don't want to wait until monday ;)

File: CHANGELOG.rst [hunk 2]
```diff
@@ -52,29 +52,6 @@ Protocol is now at version **1.12**. See `API changelog`_.
 Thanks to all contributors, with a special big-up for @gabisurita!
 
 
-4.3.4 (2016-11-17)
```
Comment (Natim) [inline CHANGELOG.rst]: Can we keep this?

File: setup.py [hunk 3]
```diff
@@ -27,7 +27,7 @@ def read_file(filename):
     'python-dateutil',
     'pyramid_multiauth >= 0.8',  # User on policy selected event.
     'transaction < 2',  # 2.X is incompatible with pyramid_tm
-    'pyramid_tm',
+    'pyramid_tm < 1.1.0',
```
Comment (Natim) [inline setup.py]: We should probably wait for 1.1.1 which will be there shortly now. https://github.com/Pylons/pyramid_tm/pull/52

File: setup.py [hunk 4]
```diff
@@ -86,7 +86,7 @@ def read_file(filename):
 
 
 setup(name='kinto',
-      version='5.0.0',
+      version='4.4.0.dev0',
```
Comment (Natim) [inline setup.py]: This should stay 5.0.0

File: CHANGELOG.rst (hunk 2 above)
Comment (leplatrem) [inline CHANGELOG.rst]: @glasserc suggested to remove them, unless I did not understand correctly

Comment (glasserc) [inline CHANGELOG.rst]: I don't feel strongly about it if @Natim wants to keep them. My real concern is just that I don't understand the workflow for releases on a release branch.

**Pull request merged**